src/lib/format/properties/parser.js

```diff
@@ -136,7 +136,7 @@ export default {
       isSimpleValue = true;
     }
 
-    if (isSimpleValue && (!entries[id] || isSimpleNode)) {
+    if (isSimpleValue && (!root[id] || isSimpleNode)) {
       if (id in root) {
         throw this.error('Duplicated id: ' + id);
```

Comment:
Or, to phrase the questions differently: is the new code capable of detecting and reporting duplicate strings in hashes?

    foo[one] = One
    foo[one] = One